### ISSUE TITLE
clarification needed in curve fitting tutorial

### ISSUE THREAD

**Issue by laserpanda** [tensorflow/tfjs-website#129]
"minimize then automatically adjusts any variables used by this function (here, the coefficients a, b, c, and d) in order to minimize the return value (our loss)."

please expand on how/why the optimizer magically knows what variables to adjust?

**Comment by tafsiri** [comment]
Hi @laserpanda, just to clarify your question, do you mean how the optimizer knows _which_ variables to update? Or how the optimizer calculates what the update should be? 

For the first question, the optimizer will update _any/all_ variables used in the function passed to it. For the second we try and explain that at the start of the __"Define the Optimizer"__ section, where we say we use Stochastic Gradient Descent. What extra detail/clarification would be helpful?

**Comment by laserpanda** [comment]
Hi
it is the first one. When i re-read it knowing what I know now it makes more sense, but automatically picking up global variables and adjusting them arbitrarily is a bit confusing the first time reading it. 

Maybe because "variables" can be interpreted as both the tensorflow variables and just generic variables. Perhaps this would be clearer?

"minimize then automatically adjusts any `Variable`s used by this function (here, the coefficients a, b, c, and d) in order to minimize the return value (our loss)."

**Comment by tafsiri** [comment]
Good suggestion! Thanks!

**Comment by tafsiri** [comment]
I think this got deployed.
